styleguide.config.js

```diff
@@ -101,7 +102,23 @@ module.exports = {
             base: '"Lato", sans-serif',
         },
     },
+    title: "Wonder Blocks",
     styles: {
+        StyleGuide: {
+            sidebar: {
+                width: "initial",
+            },
+        },
+        Logo: {
+            logo: {
+                background: "no-repeat url('logo.png')",
+                backgroundPosition: "0 0.25em",
+                paddingLeft: 40,
+                minHeight: "calc(32px + 0.5em)",
+                display: "flex",
+                alignItems: "center",
+            },
```

Comment:
This adds the logo to the title, gives it some space, and also ensures the text looks half decent alongside it. I did some testing with font-size changes and it seems to cope correctly.

Reply:
I assume this is a netlify config thing. I don't have access to that so someone else would need to look into it.

Reply:
@somewhatabstract it should work as long as the icon gets copied to the `docs` folder.

Reply:
🛑 Yeah, you should be able to update `build:styleguidist` in package.json to also copy these files over.

🛑 Also, would it be possible to put the two images into a folder named something like `static` and then copy that whole folder over? Would help to avoid cluttering the root directory.

Reply:
Ah, yeah, I can definitely do those things :)

Reply:
Well, I worked it out with only some config changes and a `static` dir. Now it works like a charm! :grin: